pkg/model/translation.go

```diff
@@ -50,6 +50,11 @@ type TranslationRule struct {
 	Resources         ResourceRequirements `json:"resources,omitempty"`
 }
 
+//IsMainDevContainer returns true if the translation rule applies to the main dev container of the okteto manifest
+func (r *TranslationRule) IsMainDevContainer() bool {
+	return r.OktetoBinImageTag != ""
```

Comment:
Could we use a label to determine this? Checking on a specific string seems brittle and hard to understand for me.

Reply:
What do you mean by label in this context?

Reply:
A kubernetes label. We could also have a flag we set in the rules object to mark it as the 'main container' when we analyze the deployment?

Reply:
This is why I refactored it as a function. But changing the representation brings issues with backward compatibility. Let's keep it like this for now since it is unrelated to this PR